code/controllers/subsystem/weather.dm

```diff
@@ -49,7 +91,7 @@ SUBSYSTEM_DEF(weather)
 			LAZYINITLIST(eligible_zlevels["[z]"])
 			eligible_zlevels["[z]"][weather] = probability
 
-/datum/controller/subsystem/weather/proc/run_weather(datum/weather/weather_datum_type, z_levels)
+/datum/controller/subsystem/weather/proc/run_weather(datum/weather/weather_datum_type, z_levels, area_type, weather_bitflags, thunder_value, datum/reagent/reagent_type)
```

Comment:
that's a LOTTTT

Reply:
This is needed so admins can pass arguments to the weather when they use their custom weather verb. The bitflags shrunk it down significantly.

Reply:
I'm going to make the arguments here into a named list

Reply:
Applied via https://github.com/tgstation/tgstation/pull/89550/commits/1c854922b9edef2dad3a53532ad6778aa2394618